web/frontend/src/components/camera/camera.svelte

```diff
@@ -14,29 +15,29 @@ const { robotClient, streamManager } = useRobotClient();
 
 let imgEl: HTMLImageElement;
 
-let cameraFrameIntervalId = -1;
-
 const cameraManager = $streamManager.setCameraManager(cameraName);
 
-const clearFrameInterval = () => {
-  window.clearInterval(cameraFrameIntervalId);
+let clearFrameInterval = () => {
+  // does nothing
 };
```

Comment:
`import { noop } from 'lodash';`

```suggestion
let clearFrameInterval = noop;
```